docs/reference/rest-api/common-parms.asciidoc

```diff
@@ -485,6 +485,12 @@ tag::query[]
 <<query-dsl,Query DSL>>.
 end::query[]
 
+tag::realtime[]
+`realtime`::
+(Optional, boolean) Set to `false` to disable real time GET
```

Comment:
For boolean parm definitions, we try to use the following convention:

If `true`, ...